dataactbroker/permissions.py

```diff
@@ -34,8 +34,9 @@ def decorated_function(*args, **kwargs):
                 elif LoginSession.isLogin(session):
                     user = sess.query(User).filter(User.user_id == session["name"]).one()
                     valid_user = True
-                    for permission in permission_list :
-                        if not has_permission(user, permission):
+                    for permission in permission_list:
+                        if permission in PERMISSION_TYPE_DICT and \
+                                not user.permission_type_id == PERMISSION_TYPE_DICT[permission]:
```

Comment:
I think we probably want to error pretty loudly if the permission specified isn't in the DICT. Maybe we do something like:
```python
def permissions_check(f=None,permission_list=[]):
    for permission in permission_list:
        if permission not in PERMISSION_TYPE_DICT:
            raise ValueError("{} not a valid permission".format(permission))
```
That'd error out at application startup (and during tests) rather than silently failing at run time.

Also, this block confuses me a bit -- if we require that a user have all of the permissions in the `permission_list`, but a user can only have one permission, why have a list?